docs/src/app/components/pages/discover-more/Showcase.js

```diff
@@ -28,6 +28,12 @@ const appList = [
   //   link: '',
   // },
   {
+    title: 'SlimChess - Play Instant Chess Online Games on the Go',
```

Comment:
I also agree on organize based on quality. I can do a quality review and reordering and submit another PR if you want.